op-acceptor/cmd/main_test.go

```diff
@@ -202,7 +202,7 @@ gates:
 			// Create temporary test directory
 			tempDir, err := os.MkdirTemp("/tmp", "op-acceptor-test-")
 			require.NoError(t, err, "Failed to create temporary directory")
-			defer os.RemoveAll(tempDir)
+			defer func() { _ = os.RemoveAll(tempDir) }()
```

Comment:
Why this change? Not that I am a go expert but this seems like just introducing a new lambda

Reply:
To appease the linter. It was complaining that we weren't checking the error code which [os.RemoveAll](https://pkg.go.dev/os#RemoveAll) returns.